source/containers.tex

```diff
@@ -25967,19 +26210,20 @@
 if for a value $u$
 for which \tcode{\exposid{rank_} - $u$ - 2}
 is the largest value p smaller than \tcode{\exposid{rank_} - 1}
-for which $S_p$ is a unit-stride slice for \tcode{mapping},
+for which \tcode{SliceSpecifiers...[$p$]} is a unit-stride slice type,
 the following conditions are met:
 \begin{itemize}
 \item
 for $k$ equal to \tcode{\exposid{rank_} - 1},
-$S_k$ is a unit-stride slice for \tcode{mapping}; and
+\tcode{SliceSpecifiers...[$k$]} is a unit-stride slice type; and
 \item
 for each $k$ in the range
+%FIXME: drive-by fix the broken parenthesis
```

Comment:
The paper asked for that change to be made, and it was in the approved wording, so please just make the requested edit.